staging/src/k8s.io/kube-aggregator/pkg/apiserver/apiserver.go

```diff
@@ -244,6 +280,20 @@ func (c completedConfig) NewWithDelegate(delegationTarget genericapiserver.Deleg
 		return nil
 	})
 
+	s.GenericAPIServer.PrepareOpenAPIService()
+
+	if s.GenericAPIServer.OpenAPIService != nil {
```

Comment:
This should fall out more cleanly once you refactor for the above comments.  I think aggregator should always server openapi, so once it's all contained in the correct repos and packages I suspect you'll end up with a cleaner end product.  This reads like coding by coincidence.